prow/cmd/clonerefs/BUILD.bazel

```diff
@@ -14,11 +15,22 @@ go_library(
     ],
 )
 
+go_image(
+    name = "clonerefs-go_image",
+    base = "@git-base//image",
+    embed = [":go_default_library"],
+    goarch = "amd64",
+    goos = "linux",
+    pure = "on",
+    visibility = ["//visibility:public"],
+)
+
 container_image(
     name = "image",
-    base = "@git-base//image",
-    directory = "/",
-    files = [":clonerefs"],
+    base = ":clonerefs-go_image",
+    symlinks = {
+        "/clonerefs": "/app/prow/cmd/clonerefs",
```

Comment:
Did we test that this actually works? I'm getting errors when trying to use these images:

```
        message: |
          oci runtime error: container_linux.go:247: starting container process caused "exec: \"/clonerefs\": permission denied"
```

Reply:
Things work again when I refer to the previous pattern.

Reply:
There was https://github.com/kubernetes/test-infra/pull/9237#issuecomment-417820937
prow.k8s.io is still on openshift's make built images

Reply:
Maybe it works if you build on linux?

Reply:
Certainly isn't working here

Reply:
yeah I'm seeing this inside the image:
```
/app # ls -la /clonerefs
lrwxrwxrwx    1 root     root            23 Jan  1  1970 /clonerefs -> /app/prow/cmd/clonerefs
/app # ls /app/prow/cmd/clonerefs
clonerefs-go_image.binary           clonerefs-go_image.binary.runfiles
```

Symlinking should work, but we'd need to get the paths right reliably:
```
/app # ln -sf /app/prow/cmd/clonerefs/clonerefs-go_image.binary /clonerefs
/app # /clonerefs
FATA[0000] Invalid options: no source root specified    
```

Reply:
Fix inbound.